### PR TITLE
[Utils] Install Script: Plugin install option

### PR DESCRIPTION
**Pull request by SAtacker**
* Added `plugins` option for installer
* Ref - https://github.com/WasmEdge/WasmEdge/issues/1606

Signed-off-by: Shreyas Atre <shreyasatre16@gmail.com>

### CODE REVIEW

**Comment by SAtacker** [comment]
```bash
curl -sSf https://raw.githubusercontent.com/SAtacker/WasmEdge/installer_plugins/utils/install.sh | bash -s -- --plugins=wasi_nn-openvino
```

**Comment by SAtacker** [comment]
@hydai It's working in my system. https://github.com/WasmEdge/WasmEdge/runs/7193455765?check_suite_focus=true#step:26:112 seems weird. Please check.

File: utils/install.sh [hunk 1]
```diff
@@ -750,6 +750,14 @@ main() {
     fi
 
     if [ $PLUGINS_INSTALL -eq 1 ]; then
+        if [ "$(printf %s\\n%s\\n "0.10.1-alpha.1" "$VERSION")" != "$(printf %s\\n%s "0.10.1-alpha.1" "$VERSION" | awk '{ if ($1 ~ /-/) print; else print $0"_" ; }' | sort --version-sort | sed 's/_$//')" ]; then
```
Comment (hydai) [inline utils/install.sh:753]: Is this limited to `Linux`? Currently, our WASI-NN with OpenVINO support is only for linux(ubuntu 20.04).

**Review by hydai — CHANGES_REQUESTED** [review]

File: utils/install.sh (hunk 1 above)
Comment (SAtacker) [reply utils/install.sh:753]: Yes please see https://github.com/WasmEdge/WasmEdge/commit/2d750c79ac7d3f3a69cab6e24e8b165b2351c930#diff-001b8193af60259ffc2829010784899f35764c864992fe3e96a606ad2650bb2bR631
Line 631

**Comment by SAtacker** [comment]
WasmEdge is not able to find the plugin installed in `$HOME/new_wasmedge` as it looks for `$HOME/.wasmedge/`

**Comment by hydai** [comment]
Hi @SAtacker 
I am going to create a new docker image for the WASI-NN plugin environment. Once it has been done, I will get back to you.

**Comment by SAtacker** [comment]
> Hi @SAtacker 
> 
> I am going to create a new docker image for the WASI-NN plugin environment. Once it has been done, I will get back to you.

Thanks, once this gets done I can start with the python installer.

**Comment by hydai** [comment]
Hi @SAtacker,

Just a quick update, instead of providing a new docker image, we are trying to make the OpenVINO build on the manylinux environment, which may need more time. So we will pause this PR until the results are out. Thanks for your contribution.

**Comment by SAtacker** [comment]
No problem, thanks for notifying me.

**Comment by hydai** [comment]
However, the python installer can be kicked off. We will have the PyTorch backend in the WASI-NN proposal soon #1654, it's totally fine to go ahead with the new installer.

**Comment by SAtacker** [comment]
> However, the python installer can be kicked off. We will have the PyTorch backend in the WASI-NN proposal soon #1654, it's totally fine to go ahead with the new installer.

That's great. I'll send a PR by tomorrow (a 24 hr period).

**Comment by hydai** [comment]
> That's great. I'll send a PR by tomorrow (a 24 hr period).

No rush :-) Take your time.

**Comment by SAtacker** [comment]
You were

> > That's great. I'll send a PR by tomorrow (a 24 hr period).
> 
> No rush :-) Take your time.

You were right, it's taking time. Especially compatibility between 2.7 and 3+.

**Comment by hydai** [comment]
Hi @SAtacker 
I believe we can close this one. Because the new installer is on the way, which will also handle the plugins.

**Comment by SAtacker** [comment]
> Hi @SAtacker 
> I believe we can close this one. Because the new installer is on the way, which will also handle the plugins.

Yes, thanks

**Pull request closed without merging**